index.html

```diff
@@ -1302,6 +1302,14 @@ <h2>Public Keys</h2>
 the <a>DID document</a>; see Section <a href="#authorization-and-delegation"></a>.
       </p>
 
+      <p class="note" title="Key Identifiers">
+The value of the <code>id</code> property may be structured as a <a href="https://en.wikipedia.org/wiki/Compound_key">compound key</a>. 
```

Comment:
Is this "may" normative? If not, replace it w/ a non-normative word... like "might", but that creates a problem w/ the SHOULD below... makes it seem less strongly suggestive.

Reply:
can a note contain such language? I will make it normative, I'd prefer it to be.

Reply:
Yeah, good point. Typically, no, a note shouldn't contain normative language. I've seen some specs sneak a couple of those in. 

If it were me, I'd remove `class="note"` -- this is normative and it should clearly state that.

Reply:
Can I resolve this?